### ISSUE TITLE
DAP violations in OverflowMenu component

### ISSUE THREAD

**Issue by prince737** [carbon-design-system/carbon#5526]
The `OverflowMenu` component shows the following violations:
- Accessible name does not match or contain the visible label text.

## Environment
carbon-components: 10.10.1
carbon-components-react: 7.10.1
DAP Jan 2020 Ruleset

## Sandbox
https://codesandbox.io/s/codesandbox-x8m32

## Screenshots 
<img width="1428" alt="Screenshot 2020-03-04 at 11 56 05 AM" src="https://user-images.githubusercontent.com/31027320/75851259-29618a00-5e0f-11ea-8323-f62de882e550.png">

**Comment by asudoh** [comment]
Hi 👋 the error is saying that the content of icon (specifically `iconDescription` in your test case) does not match `aria-label`. Hope this helps.

**Comment by prince737** [comment]
@asudoh In the same sandbox now getting `Orphan content found that does not reside within a WAI-ARIA landmark or labelled region role.` violation when the menu is expanded. Can you please have a look. Just wanted to make sure it was not something silly like this before creating an issue.

<img width="1393" alt="Screenshot 2020-03-04 at 6 44 54 PM" src="https://user-images.githubusercontent.com/31027320/75883042-5c743f80-5e48-11ea-922b-a0cb7e06c5c2.png">

**Comment by asudoh** [comment]
You can set `data-floating-menu-container` attribute to a DOM node in a region DAP wants the contents in.
